### PR TITLE
Neat sea 1: Loader

### PR DESCRIPTION
**Pull request by modernserf**
http://neat-sea-1.glitch.me/

- @ehmorris: Is this correctly proportioned to its surrounding text? We rarely use loaders inline on community. Also, any thoughts on what to do in the 'prefer-reduced-motion` case?

### CODE REVIEW

File: lib/icon.js [hunk 1]
```diff
@@ -61,16 +60,28 @@ const svgs = {
       <polygon fill="currentColor" points="8.5,3 8.5,9 4.5,6" />
     </SVGBase>
   ),
-  eye: (props) => (  
+  eye: (props) => (
```
Comment (modernserf) [inline lib/icon.js]: it appears that i dropped a commit on the Icon PR

**Comment by modernserf** [comment]
Updated Loader to use 'size' prop, though now I'm not sure if using text sizes makes sense for either icons or loaders -- maybe this scale should be linear (e.g.`8px 16px 24px 32px...`) or even exponential (e.g. `8px 16px 32px 64px ...`)

**Comment by modernserf** [comment]
Another option to consider: you can also override the sizes on the component directly, without a wrapper, like 

```js
const BigIcon = styled(Icon)`
  font-size: 100px;
`
```

That _does_ require a special case, but it doesn't require a wrapper div; that might be the right approach if we don't know what icon/loader/etc sizes we'll need in advance.

**Comment by ehmorris** [comment]
I do think the sizes look correct in your demo. We use some larger-than-inline inline loaders in the editor, but I think these would be an improvement.

I still think font-size is a bit unintuitive for setting the size for icons and loaders. I'd expect to be able to set a width or height, and have the icon or loader react like a normal svg or image. Allowing overrides like you show is really nice.

As for reduced motion, I'm not sure what we could do without doing something totally different. Would we be able to remove some but not all of the animating elements? Maybe we could remove the "dust" layers and just slowly animate the larger circles past each other.

**Comment by modernserf** [comment]
- updated icon and loader to remove size prop in favor of overriding with inline styles / styled components
-  created 'reduced motion' animation (just fading between pink and black)

**Comment by keithk** [comment]
:rocket: PR was released in [`v0.19.0`](https://github.com/glitchdotcom/shared-components/releases/tag/v0.19.0) :rocket:

**Pull request merged**